conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/feignimpl/InputStreamDelegateEncoder.java

```diff
@@ -52,7 +51,7 @@ public InputStreamDelegateEncoder(@Safe String clientNameForLogging, Encoder del
     public void encode(Object object, Type bodyType, RequestTemplate template) throws EncodeException {
         if (bodyType.equals(InputStream.class)) {
             try {
-                byte[] bytes = Util.toByteArray((InputStream) object);
+                byte[] bytes = ((InputStream) object).readAllBytes();
```

Comment:
This may expose resource leaks since it no longer closes the input, we should update this to preserve existing behavior™